Build fails on mac

```
master~/.vim/plugged/cpsm$ ./install.sh
PY3 not specified; inferring Python version from /usr/local/bin/vim
Python 3 selected
-- The C compiler identification is AppleClang 9.1.0.9020039
-- The CXX compiler identification is AppleClang 9.1.0.9020039
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - found
-- Found Threads: TRUE
-- Boost version: 1.67.0
-- Found the following Boost libraries:
--   program_options
-- Found PythonInterp: /usr/local/bin/python3 (found suitable version "3.7", minimum required is "3")
-- Found PythonInterp: /usr/local/bin/python3 (found version "3.7")
-- Found PythonConfig: -I/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/include/python3.7m -I/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/include/python3.7m; -L/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/config-3.7m-darwin -lpython3.7m -ldl -framework CoreFoundation
-- Optional package ICU was not found
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/xxx/.vim/plugged/cpsm/build
Scanning dependencies of target cpsm_core
[ 11%] Building CXX object CMakeFiles/cpsm_core.dir/src/str_util.cc.o
[ 22%] Linking CXX static library libcpsm_core.a
[ 22%] Built target cpsm_core
Scanning dependencies of target cpsm_cli
[ 33%] Building CXX object CMakeFiles/cpsm_cli.dir/src/cpsm_cli_main.cc.o
[ 44%] Linking CXX executable cpsm_cli
[ 44%] Built target cpsm_cli
Scanning dependencies of target cpsm_py
[ 55%] Building CXX object CMakeFiles/cpsm_py.dir/src/ctrlp_util.cc.o
[ 66%] Building CXX object CMakeFiles/cpsm_py.dir/src/python_extension.cc.o
/Users/xxx/.vim/plugged/cpsm/src/python_extension.cc:48:11: error: assigning to 'char *' from incompatible type 'const char *'
  *data = PyUnicode_AsUTF8AndSize(obj, size);
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/cpsm_py.dir/src/python_extension.cc.o] Error 1
make[1]: *** [CMakeFiles/cpsm_py.dir/all] Error 2
```

--- a/src/python_extension.cc
+++ b/src/python_extension.cc
@@ -42,7 +42,7 @@ typedef std::unique_ptr<PyObject, PyObjectDeleter> PyObjPtr;
 
 // Wrappers around Python 2/3 string type distinctions.
 
-inline bool PyVimString_AsStringAndSize(PyObject* obj, const char** data,
+inline bool PyVimString_AsStringAndSize(PyObject* obj, char** data,
                                         Py_ssize_t* size) {
 #if PY_MAJOR_VERSION >= 3
   *data = (char*)PyUnicode_AsUTF8AndSize(obj, size);
@@ -106,7 +106,7 @@ class PyIterCtrlPMatchSource {
       if (item_obj == nullptr) {
         return false;
       }
-      const char* item_data;
+      char* item_data;
       Py_ssize_t item_size;
       if (!PyVimString_AsStringAndSize(item_obj.get(), &item_data,
                                        &item_size)) {
@@ -163,7 +163,7 @@ class PyListCtrlPMatchSource {
       if (item_obj == nullptr) {
         return false;
       }
-      const char* item_data;
+      char* item_data;
       Py_ssize_t item_size;
       if (!PyVimString_AsStringAndSize(item_obj, &item_data, &item_size)) {
         return false;

--- a/src/python_extension.cc
+++ b/src/python_extension.cc
@@ -42,10 +42,10 @@ typedef std::unique_ptr<PyObject, PyObjectDeleter> PyObjPtr;
 
 // Wrappers around Python 2/3 string type distinctions.
 
-inline bool PyVimString_AsStringAndSize(PyObject* obj, char** data,
+inline bool PyVimString_AsStringAndSize(PyObject* obj, const char** data,
                                         Py_ssize_t* size) {
 #if PY_MAJOR_VERSION >= 3
-  *data = PyUnicode_AsUTF8AndSize(obj, size);
+  *data = (char*)PyUnicode_AsUTF8AndSize(obj, size);
   return *data != nullptr;
 #else
   return PyString_AsStringAndSize(obj, data, size) >= 0;
@@ -106,7 +106,7 @@ class PyIterCtrlPMatchSource {
       if (item_obj == nullptr) {
         return false;
       }
-      char* item_data;
+      const char* item_data;
       Py_ssize_t item_size;
       if (!PyVimString_AsStringAndSize(item_obj.get(), &item_data,
                                        &item_size)) {
@@ -163,7 +163,7 @@ class PyListCtrlPMatchSource {
       if (item_obj == nullptr) {
         return false;
       }
-      char* item_data;
+      const char* item_data;
       Py_ssize_t item_size;
       if (!PyVimString_AsStringAndSize(item_obj, &item_data, &item_size)) {
         return false;

--- a/src/python_extension.cc
+++ b/src/python_extension.cc
@@ -45,7 +45,7 @@ typedef std::unique_ptr<PyObject, PyObjectDeleter> PyObjPtr;
 inline bool PyVimString_AsStringAndSize(PyObject* obj, char** data,
                                         Py_ssize_t* size) {
 #if PY_MAJOR_VERSION >= 3
-  *data = (char*)PyUnicode_AsUTF8AndSize(obj, size);
+  *data = const_cast<char *>(PyUnicode_AsUTF8AndSize(obj, size));
   return *data != nullptr;
 #else
   return PyString_AsStringAndSize(obj, data, size) >= 0;

